boilerplate/app/app.tsx.ejs

```diff
@@ -49,10 +56,21 @@ const canExit = (routeName: string) => contains(routeName, exitRoutes)
 /**
  * This is the root component of our app.
  */
+<% if (!props.useExpo) { %>
 export const App: React.FunctionComponent<{}> = () => {
+<% } else { %>
+export default function App() {
+<% } %>
   const [rootStore, setRootStore] = useState<RootStore | undefined>(undefined) // prettier-ignore
   useEffect(() => {
+<% if (props.useExpo) { %>
+    (async () => {
+      await initFonts()
+      setupRootStore().then(setRootStore)
+    })()
+<% } else { %>
     setupRootStore().then(setRootStore)
+<% } %>
```

Comment:
Simplified:

```suggestion
  (async () => {
    <% if (props.useExpo) { %>
      await initFonts()
    <% } %>
    setupRootStore().then(setRootStore)
  })()
```

Reply:
I hard-separated because it might look confusing to guess the intention for RN ver. but will apply this.

Reply:
I see, yeah, makes sense.